js/plates-model/generate-plates.js

```diff
@@ -32,10 +32,10 @@ export default function generatePlates (imgData, initFunction) {
     }
     plates[key].addField({ id: fieldId, age: MAX_AGE, type, elevation, crustThickness })
   })
-  Object.values(plates).forEach(plate => plate.updateInertiaTensor())
+  Object.keys(plates).map(key => plates[key]).forEach(plate => plate.updateInertiaTensor())
   // User-provided function that can modify default options of all the plates.
   if (initFunction) {
     initFunction(plates)
   }
-  return Object.values(plates)
+  return Object.keys(plates).map(key => plates[key])
```

Comment:
Tests were failing for me locally when using Object.values, so I swapped it out here. Apparently Object.values doesn't work on IE either, so this may be preferred

Reply:
I guess you might be using pretty old Node.js version. And the code is transpiled by Babel, so my guess is that Object.values gets replaced with something else or there's polyfill. But that's fine anyway. 👍